launchdarkly/README.md

```diff
@@ -1,45 +1,107 @@
 # LaunchDarkly Integration
 
-## Overview
+There are currently two integrations with LaunchDarkly:
 
-This integration tracks metrics from [LaunchDarkly's relay proxy][1], such as number of proxied stream connections and proxied API route requests.
+- [Events Integration](#launchdarkly-events-integration)
+- [LD Relay Integration](#ld-relay-integration)
 
-## Setup
+## LaunchDarkly Events Integration
 
-### Configuration
+### Overview
 
-After you've configured the [relay proxy][2], add the following section to your LaunchDarkly relay instance's `ld-relay.conf` file:
+Deploying features with LaunchDarkly is great, but sometimes it's difficult to
+see the effects of these feature deployments on the customer's services or
+systems. Datadog's monitoring and analytics tools for modern DevOps teams allow
+their customers to observe the health of their systems over time. The
+LaunchDarkly integration with Datadog exposes the LaunchDarkly events in context
+with events gathered by the Datadog agents. This makes it easier to reason about
+changes to customers' systems as a result of feature deployments that took
+place. For example, if a feature gets deployed that causes a service to slow
+down, a DevOps engineer will now be able to see the cause within Datadog.
+
+![LaunchDarkly events in Datadog][7]
+
+### Setup
+
+#### Configuration
+
+As a prerequisite, this integration will need a [Datadog API
+key][6]. Note, only Admin users can create an API key. Once you've obtained a Datadog API key, visit the [LaunchDarkly documentation for the Datadog integration][8].
+
+### Data Collected
+
+#### Metrics
+
+The LaunchDarkly integration does not include any metrics.
+
+#### Events
+
+This integration sends LaunchDarkly flag, project, environment events to Datadog. These events can be used to correlate with other metrics available in Datadog.
+
+#### Service Checks
+
+The LaunchDarkly integration does not include any service checks.
+
+### Troubleshooting
+
+Need help? Contact [LaunchDarkly Support][9].
+
+### Further Reading
+
+Learn more about [LaunchDarkly] and this integration [here][8].
+
+---
+
+## LD Relay Integration
```

Comment:
This is a little inconsistent, but even though the feature is called the LD Relay, we refer to it in docs as the "relay proxy." Update the instances in the text below where you refer to it as the "relay" to "relay proxy," please. :)